charts/rancher-vsphere-cpi/Chart.yaml

```diff
@@ -5,10 +5,10 @@ annotations:
   catalog.cattle.io/namespace: kube-system
   catalog.cattle.io/os: linux
   catalog.cattle.io/permits-os: linux,windows
-  catalog.cattle.io/rancher-version: '>= 2.6.0-0'
+  catalog.cattle.io/rancher-version: '>= 2.8.0-0'
```

Comment:
This is a bit out of the scope but looking into this change got me thinking.
`rancher/vsphere-charts` is base (super) chart that is then consumed by both Rancher (of various minor release lines) and RKE2 (I'm assuming RKE2 removes this annotation (or should remove it)).

So I'm wondering if we should either:
a.) Altogether remove this line and have "consumers" of this chart (like various branches in `rancher/charts`) add it through patch(diff) file
b.) Have a placeholder value like '>= PLACEHOLDER-RANCHER-VERSION' to prevent instances of forgetting to set it.

@a-blender @brandond , WDYT?

Reply:
It's interesting.. sometimes we forget to update `rancher-version` in vsphere-charts, so we patch it in the rancher/charts PR so we've done a) arguably several times already. I'd just keep doing that and remove the line in vsphere charts imo.

Reply:
It's just an annotation, and the RKE2 HelmChart controller doesn't make use of any of the rancher catalog annotations, so it's harmless to have it here as far as as I'm concerned. If it's easier to maintain it here then I'm fine with leaving it.